docker-compose.yml

```
@@ -1085,3 +1121,5 @@
     driver: local
   azurite_data:
     driver: local
+  terminology_data:
+    driver: local
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)